pontoon/homepage/templates/home.html

```diff
@@ -0,0 +1,175 @@
+{% extends 'base.html' %}
+
+{% block title %}Localize Mozilla{% endblock %}
+
+{% block class %}home{% endblock %}
+
+{% block middle %}
+<section id="main">
+  <div>
+
+    <!-- Pagination -->
+    <ul class="side-navigation">
+      <li class="pagination active" href="#section-1"></li>
+      <li class="pagination" href="#section-2"></li>
+      <li class="pagination" href="#section-3"></li>
+      <li class="pagination" href="#section-4"></li>
+      <li class="pagination" href="#section-5"></li>
+      <li class="pagination" href="#section-6"></li>
+    </ul>
+
+    <!-- Section-1: Introduction -->
+    <section class="snaps section-1 relative" data-section-name="section-1">
+      <div class="introduction neg-margin--large">
+        <div class="title-style">A place to localize Mozilla, fast and easy.</div>
+        <div class="content-style">Join a global community of thousands in making Firefox and various Mozilla products speak to over a hundred of cultures in their native language.</div>
+        <div class="flex vertical-align-portrait">
+          <a class="homepage-button primary-button-cta" href="{{ url('pontoon.teams') }}">
+            <div>Start Localizing Now</div>
+          </a>
+          <p>or</p>
+          <a class="homepage-button secondary-button-cta" href="/tour">
```

Comment:
Hardcoded relative URLs in templates are forbidden.

Instead of `/tour`, you should use this:
`{{ url('pontoon.translate.locale.agnostic', 'projects', 'demo', 'all-resources') }}`

Instead of `/in-context`, you should use this:
`{{ url('pontoon.translate.locale.agnostic', 'projects', 'pontoon-intro', 'messages.properties') }}`

Instead of `/static/img/image-name.svg`, you should use this:
`{{ static('img/image-name.svg') }}`